releasenotes/notes/actionset-podlabels-annotations-915f1dfa7ee86978.yaml

```
@@ -0,0 +1 @@
+features: Added two ActionSet fields ``podLabels`` and ``podAnnotations`` that can be used to configure the labels and annotations of the Kanister function pod, run by an ActionSet.
```

```suggestion
features: Added two new fields, ``podLabels`` and ``podAnnotations``, to the ActionSet. These fields can be used to configure the labels and annotations of the Kanister function pod run by an ActionSet.
```
[nit] to improve readability